source/containers.tex

```diff
@@ -9261,8 +9261,6 @@
 
     // bit reference
     class @\libmember{reference}{vector<bool>}@ {
-      constexpr reference() noexcept;
```

Comment:
For the record, the friend declarations were removed by #6427

Reply:
Thanks, appreciated!